build.gradle

```diff
@@ -49,6 +49,7 @@ plugins {
     id("org.nosphere.apache.rat") version "0.8.0"
 
     id 'jacoco'
+    id("io.github.surpsg.delta-coverage") version "2.4.0"
```

Comment:
we need to upgrade gradle for the project :)

Reply:
Agreed! I'll create a Jira for it ☺️